.tekton/on-pull-request.yaml

```diff
@@ -151,16 +173,16 @@ spec:
                   mountPath: $(workspaces.source.path)/examples
```

Comment:
@vbelouso Not related to this PR, but i noticed by chance that if the configmap `integration-tests-reports` volume mount into this path will be deleted from the namespace, then it will crash the pipeline. and we are not even started to implement integration tests for ExploitIQ, and this is a potential point of failure for something not being used currently....
Isn't it better to:
1. remove it in the meantime, at least until we're going to implement integration test.
2. Or not removing it, and just put the integration tests reports  in an examples dir inside the  repository'  , and then to take that examples dir with reports essential for IT from the `source` workspace .

Reply:
Hey @zvigrinberg 
I don't mind removing everything related to the integration tests entirely in this PR. I mean all the mounts and script changes.

Reply:
Hi @vbelouso 
Yes i believe that would be best.